index.js

```diff
@@ -4,7 +4,20 @@ module.exports = {
         MyGlobal: true
```

Comment:
hm, is this `MyGlobal` global really used somewhere? 😄

Reply:
nope :smile: